v2/emailpassword/serverless/with-aws-lambda/authorizer.mdx

```diff
@@ -149,20 +149,20 @@ def handler(event: Dict[str, Any], context: Any):
 
 Use the code below as the handler for the lambda. Remember that whenever we want to use any functions from the `supertokens-node` lib, we have to call the `supertokens.init` function at the top of that serverless function file. We can then use `getSession()` to get the session.
 
-```tsx title="auth.ts"
+```tsx title="index.mjs"
 import supertokens from "supertokens-node";
 import { SessionEvent } from "supertokens-node/framework/awsLambda";
 import { APIGatewayAuthorizerEvent, PolicyDocument, Statement, AuthResponse } from "aws-lambda";
 import Session from "supertokens-node/recipe/session";
 
 // @ts-ignore
-import { getBackendConfig } from "./config";
+import { getBackendConfig } from "./config.mjs";
 
 supertokens.init(getBackendConfig());
 
 type AuthorizerEvent = SessionEvent & APIGatewayAuthorizerEvent;
 
-exports.handler = async function (event: AuthorizerEvent) {
+export const handler = async function (event: AuthorizerEvent) {
```

Comment:
should you use the OMIT_START thing here too? Since .mjs don't have types? Please check all files and snippets once again

Reply:
Done!


Let me know what you think about changing `OMIT_START` to `OS` and `OMIT_END` to `OE`. It will be make things concise but less descriptive. It may be fine to leave it as is because I don't see us using it a lot currently.